### PR TITLE
feat: Added Options Button in Docs page.

### PR DESCRIPTION
**Pull request by adity2911**
## Related Issue
#1519

## Description
I have added the button that partitions the two methods of showing how to contribute to the user.

## Screenshots
image

### CODE REVIEW

**Comment by priyankarpal** [comment]
will check soon

File: app/docs/page.tsx
Comment (priyankarpal) [inline app/docs/page.tsx:1]: line no. `164` just add only `cursor-pointer text-indigo-600` & remove other tailwind CSS classes

Comment (priyankarpal) [inline app/docs/page.tsx:1]: line no `174` 

update the code to 
```
<SyntaxHighlighter language="nginx" style={anOldHope}>
```

Comment (priyankarpal) [inline app/docs/page.tsx:1]: line no `179` modify to 
```
 <Image src={image} alt={text} className="mt-2" width={10000} height={600} />
```

File: app/docs/page.tsx [hunk 1]
```diff
@@ -235,12 +232,27 @@ function AddYourProjectsGuide(): any {
             />
           </svg>
         </div>
+        <div className="flex justify-center mt-10">
+          <button
+            className={`mx-2 px-4 py-2 rounded-lg mb-4 ${selectedOption === 0 ? "bg-indigo-600 text-white animate-pulse" : "bg-gray-800 text-gray-300"
```
Comment (priyankarpal) [inline app/docs/page.tsx:237]: modify the code to 

``` suggestion
            className={`mx-2 px-4 py-2 rounded-lg mb-4 ${selectedOption === 0 ? "bg-indigo-600 text-white" : "border border-gray-700 text-gray-300"
```

File: app/docs/page.tsx [hunk 2]
```diff
@@ -235,12 +232,27 @@ function AddYourProjectsGuide(): any {
...
+              }`}
+            onClick={() => selectOption(0)}
+          >
+            Add Projects Directly From GitHub
+          </button>
+          <button
+            className={`mx-2 px-4 py-2 rounded-lg mb-4 ${selectedOption === 1 ? "bg-indigo-600 text-white animate-pulse" : "bg-gray-800 text-gray-300"
```
Comment (priyankarpal) [inline app/docs/page.tsx:244]:
```suggestion
            className={`mx-2 px-4 py-2 rounded-lg mb-4 ${selectedOption === 1 ? "bg-indigo-600 text-white" : "border border-gray-700 text-gray-300"
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by adity2911** [comment]
@priyankarpal The reviewed changes have been resolved.

File: app/docs/page.tsx
Comment (priyankarpal) [inline app/docs/page.tsx:1]: line no `144` 
```
Cannot set properties of null (setting 'innerText')
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by adity2911** [comment]
Okay let me look into it.

**Comment by adity2911** [comment]
Resolved.

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Pull request merged**